### ISSUE TITLE
Remove jupyter.useNotebookEditor setting

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode-jupyter#5130]

**Comment by IanMatthewHuff** [comment]
Validated. Setting is gone now.

**Comment by Pugio** [comment]
Is there an alternative to this? I can't seem to find a way to automatically open '.ipynb' as JSON anymore. Sometimes I just want to see the raw file, and not start a whole notebook editing instance.

**Comment by gchenfc** [comment]
@joyceerhl @IanMatthewHuff Could one of you clarify? I'm not clear on where justifications for planned changes are documented (this issue doesn't have a description?) and I would really like to know of any alternatives.

It's quite frustrating for .ipynb files to automatically open up in notebook form without an option to view them in json form, especially for diff in source control.  And sometimes when the .ipynb file is broken for whatever reason (e.g. because source control did a bad merge because we can't view the file in json form), it can't be fixed in vscode since the jupyter viewer just crashes (I renamed the file to .json, edited, then renamed it back, then repeated until all the mistakes were fixed).

Combined with the fact that I can't seem to uninstall the Jupyter extension without uninstalling the entire Python extension, this is kind of a big inconvenience and I bet a pretty decent chunk of the python coding world might have the same issue?
